test/e2e/metering_manual_install_test.go

```diff
@@ -107,6 +114,105 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+	//Waiting for Metering pods
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	_, err := createTestingNamespace(ctx.Client, "nfsTestingNamespace")
+	if err != nil {
+		return err
+	}
+
+	files := map[string]string{
+		"pod":          "/test/e2e/manifest/nfs/nfs-server.yaml",
+		"service":      "/test/e2e/manifest/nfs/nfs-service.yaml",
+		"storageClass": "/test/e2e/manifest/nfs/nfs-storageclass.yaml",
+	}
+	server := &corev1.Pod{}
+	service := &corev1.Service{}
+	storageClass := &storagev1beta1.StorageClass{}
+	for name, file := range files {
+
```

Comment:
Let's just build up the path to these manifests here.
```suggestion
absFile := filepath.Join(repoPath, file)
```
And then we can use that `absFile` value in the call to deploy.DecodeYAMLManifestsToObject. For example:

```go
deploy.DecodeYAMLManifestToObject(absFile, server)
```